client/my-sites/plans-features-main/hooks/use-generate-action-hook.tsx

```diff
@@ -256,8 +255,7 @@ function useGenerateActionHook( {
 			text = translate( 'Contact support', { context: 'verb' } );
 			status = 'disabled';
 
-			// TODO: Consider DRYing this up
-			if ( sitePlanSlug === planSlug && plansIntent !== 'plans-p2' ) {
```

Comment:
If we don't care about `plans-p2`, then let's also remove the (equivalent?) check in `actions.tsx`? 
I'll drop a comment/suggestion below.

Reply:
I'll remove it and double check the p2 flow 👍

Reply:
[07bdebc](https://github.com/Automattic/wp-calypso/pull/90611/commits/07bdebc7148a8640d762f3c1f5df01448a5fbda3)